lib/vc.js

```diff
@@ -91,20 +98,25 @@ module.exports = {
 async function issue(options = {}) {
   const documentLoader = options.documentLoader || defaultDocumentLoader;
 
-  const {controller, domain} = options;
+  const {controller, date, maxTimestampDelta, suite} = options;
   const purpose = options.purpose || new CredentialIssuancePurpose({
-    controller,
-    domain
+    controller, date, maxTimestampDelta
   });
 
+  // check to make sure the `suite` has required params
+  // Note: verificationMethod defaults to publicKey.id, in suite constructor
+  if(!suite || !suite.verificationMethod) {
+    throw new TypeError('"suite.verificationMethod" property is required.')
+  }
+
   // run common credential checks
-  const {credential} = options;
+  const credential = JSON.parse(JSON.stringify(options.credential)); // clone
   if(!credential) {
```

Comment:
This check no longer works after the clone, the `JSON.parse` will throw if `options.credential` is not defined. Why the clone?

Reply:
Removed, thanks.